test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -69,5 +64,21 @@ describe('distribute with fractions', function () {
     ],
   ];
 
-  tests.forEach(t => testDistributeResult(t[0], t[1]));
+  const multiStepTests = [
+
+    ['(2 / x +  3x^2) * (x^3 + 1)',
+      ['(2 * (x^3 + 1) / x + 3x^2 * (x^3 + 1))',
+        '(2 * (x^3 + 1) / x + (3x^5 + 3x^2))']
+    ],
+
+    ['(2x + x^2) * (1 / (x^2 -4) + 4x^2)',
```

Comment:
Okay, I added that as a test (which does end up getting multiplied out). I think the existing test not getting fully distributed is due to the `1` being a constant rather than say `2x`? If we wanted a quick fix for that test, we could just add a rule that `1 * (any expr) = (any expr)`. What do you think?